newcomers-tutorial.md

```diff
@@ -246,42 +240,39 @@ can specify multiple tests, separated by whitespace:
 
 ## Finding a task to work on
 
-Since you came here, you probably have an idea what to work on already - but if
-not, do check the [issues tagged
+You probably have an idea what you want to work on already - but if not,
+do check the [issues tagged
 "newcomer"](https://gitlab.haskell.org/ghc/ghc/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name[]=newcomer) -
 these are issues we expect to be "low hanging fruit".
-Of course, we can't ever be sure of how hard a task is before doing it, so
-apologies if one of these is too hard.
+
+Of course, we can't be sure how hard a task is before doing it, so
+apologies in advance if one of these is too hard.
 
 Either way, **if you are going to work on something, make sure a ticket exists
 for it**. This is essential for all sorts of things, but most importantly,
 attaching things to tickets makes sure they don't get lost. So if there is no
-ticket for it yet, do file one. The issue tracker is [right
-here](https://gitlab.haskell.org/ghc/ghc/issues/new).
+ticket for it yet, please use the [issue tracker](https://gitlab.haskell.org/ghc/ghc/issues/new) to do so.
```

Comment:
Good, but keep line lengths consistent if possible, and ditch the "to do so" part, or maybe reword as "..., please create one on the [issue tracker](...)".